src/Commands/Site/GetSiteLevelFileVersionExpirationReportJobProgress.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.SharePoint.Client;
+
+using PnP.Framework.Utilities;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+
+using System.Management.Automation;
+
+namespace PnP.PowerShell.Commands.Sites
+{
+    [Cmdlet(VerbsCommon.Get, "SiteLevelFileVersionExpirationReportJobProgress")]
+    [OutputType(typeof(string))]
+    public class GetSiteLevelFileVersionExpirationReportJobProgress : PnPSharePointCmdlet
+    {
+        [Parameter(Mandatory = true)]
+        public string ReportUrl;
+
+        protected override void ExecuteCmdlet()
+        {
+            var site = ClientContext.Site;
+            var status = site.GetProgressForFileVersionExpirationReport(ReportUrl);
```

Comment:
Can you provide a sample of what this will return? I believe to have seen JSON being returned before. Would that be the same here? Or is it really a simple text like "InProgress", "Completed" and such?

Reply:
Ah we were thinking the same thing about the clarity on the return value. I had just pushed a new commit adding an object for the returned result :) Does that help?